spec/factories.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :book do
+    genre Faker::HitchhikersGuideToTheGalaxy.location
```

Comment:
For factories with Faker we used to use block, this allow factory to not have static value during all the testing process.
To able this behaviour define your faker in `{ }`
like:
```ruby
genre     { Faker::HitchhikersGuideToTheGalaxy.location }
author    { Faker::Name.name }
publisher { Faker::Name.name }
# ...
```
And even better if you align each block as above.